frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -457,6 +467,279 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [LSO_DEVICE_DISCOVERY, OCS_ATTACHED_DEVICES_FLAG],
     },
   },
+  // Noobaa Related Plugins
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: `/k8s/ns/:ns/${ClusterServiceVersionModel.plural}/:appName/${referenceForModel(
+        models.NooBaaBucketClassModel,
+      )}/~new`,
+      loader: () =>
+        import('./components/bucket-class/create-bc' /* webpackChunkName: "create-bc" */).then(
+          (m) => m.default,
+        ),
+    },
+    flags: {
+      required: [NOOBAA_FLAG],
```

Comment:
Shouldn't this be `CEPH_FLAG` ? or is this nooba specific ?

Reply:
This is noobaa specific.